zutil.h

```diff
@@ -248,6 +248,8 @@ void ZLIB_INTERNAL   zng_cfree(void *opaque, void *ptr);
 #  include "arch/x86/x86.h"
 #elif defined(ARM_CPUID)
 #  include "arch/arm/arm.h"
+#elif defined(POWER_FEATURES)
```

Comment:
Perhaps name this `POWER_CPUID` along with the rest? Maybe at a later stage we can rename these all `_FEATURES`..

Reply:
@nmoinvaz I thought about it, but POWER does not have a `cpuid`-like instruction/register, so I thought it would be somewhat confusing, so I ended up using `POWER_FEATURES` instead.

Reply:
@Dead2 might have some input on it but we should make it consistent eventually.

Reply:
Making the defines consistent should be separate pull request however. I have no personal preference if it should be applied before or after this pull request.

Reply:
ARM also calls them features, do they not?
I am not sure it really matters much though.

Reply:
@Dead2 Yup... There is no many different ARM cpu names because same CPU can have different features depending on manufacturer's choice....